velox/functions/prestosql/URLFunctions.h

```diff
@@ -84,6 +107,37 @@ FOLLY_ALWAYS_INLINE void urlEscape(TOutString& output, const TInString& input) {
   output.resize(outIndex);
 }
 
+/// Performs basic initial validation of the URI.
+/// Checks if the URI contains ascii whitespaces or
```

Comment:
I wonder if this check is complete? Are all ASCII characters other than whitespace allowed?

Reply:
Yes ascii characters other than whitespaces are allowed.